serializer.go

```diff
@@ -264,6 +274,18 @@ func getIDorName(v map[string]any) (id string, name string) {
 	return
 }
 
+func getExistsCause(v map[string]any) (exists bool, cause string) {
+	if existsRaw, hasExists := v["exists"]; hasExists {
+		if exists = existsRaw.(bool); exists == false {
```

Comment:
it's already a boolean :)
```suggestion
		if exists = existsRaw.(bool); !exists {
```